rest/src/main/kotlin/builder/component/SelectMenuBuilder.kt

```diff
@@ -16,9 +17,10 @@ import kotlin.contracts.contract
  * @param customId The identifier for the menu, max 100 characters.
  */
 @KordDsl
-public class SelectMenuBuilder(
+public open class SelectMenuBuilder(
     public var customId: String,
 ) : ActionRowComponentBuilder() {
+    internal open val type: ComponentType = ComponentType.StringSelect
```

Comment:
```suggestion
    internal open val type: ComponentType get() = ComponentType.StringSelect
```
can be a getter, so no field is generated, same for subtypes